posts/innovative-snowflake-features.md

```diff
@@ -0,0 +1,208 @@
+
+## Warehouse Data Cache
+**Implemented in the Virtual Warehouse Layer**
+
+All Snowflake Virtual Warehouses have attached SSD Storage. This SSD Storage is used to store micro-partitions that have been pulled from the Storage Layer. Reading from SSD is faster. As such, when a warehouse receives a query to process, it will first scan the SSD cache for received queries, then pull from the Storage Layer.
+
+The SSD Cache stores query specific FILE HEADER and COLUMN data. This cache type has a finite size and uses the LRU (Least Recently Used) policy to purge data that has not been recently used.
+
+---
+# Data Sharing
+Snowflake currently supports data-sharing between accounts in the same geographic region.
+
+## Data Sharing Accounts
+* Data Providers - Share the data. Providers own the data and get billed for its' storage.
+* Data Consumers - Can access and query objects in shared data. Consumers pay for compute on shared resources.
+* Reader Account - Consumer without a Snowflake Account or for Accounts not in the same geographic region as the provider. These accounts are created and completely paid for by the Providers. Reader Account Consumers can only read data shared with them.
+
+###Data Sharing for HIPAA Protected Datasets
+Data in an Enterprise for Sensitive Data (ESD) Account which is HIPAA protected can only be shared with other ESD accounts.
+
+## Sharing Data
+Snowflake uses shares, which are Snowflake objects containing all the information needed to share objects within a database. Shares contain privileges that grant access to the database, schema, or specific objects as well as the Consumer Account Name.
+
+Shares can only currently be created for other accounts in our region. You receive a share as a consumer by creating a database for it either through the Web UI or SnowSQL.
+
+---
+# Supported Data Formats and Types
+##Semi-Structured Data
+Snowflake natively supports the load and access of several types of Semi-Structured data, including JSON, Avro, XML and Parquet.
+
+In order to support loading these data-types, Snowflake has a few specialized data-types. These are:
+* VARIANT - Universal type that can store values of any other type.
+* ARRAY
+* OBJECT - Collection of key-value pairs where the key is a non-empty string and the value is of VARIANT type.
+
+### Accessing Values From JSON
+It is possible to access values from JSON in the two ways below:
+
+```plsql
+SELECT v['key1'] FROM (SELECT PARSE_JSON('{"key1":"value1", "key2":2}') as v);
+```
+OR
+```plsql
+SELECT v:key1 FROM (SELECT PARSE_JSON('{"key1":"value1", "key2":2}') as v);
+```
+
+In addition, if required, data can becast from Variants into SQL types using the **::** operator.
+
+### Accessing Data in Arrays (JSON)
+LATERAL FLATTEN() is the function Snowflake provides for accessing data from nested arrays. [For more Information And a Tutorial Click Here!](https://community.snowflake.com/s/article/How-To-Lateral-Join-Tutorial)
+
```

Comment:
You should add a paragraph at the very end summarizing what you told them and a possible call-to-action.  If they are interested, Ippon can assist.